backend/controllers/menu.js

```diff
@@ -6,11 +6,7 @@ const menuFactory = (prisonApi, logError) => {
       res.render('menu.njk', {
         title: 'Manage user accounts',
         homeUrl: config.app.notmEndpointUrl,
-        contentUrl: `${config.app.dpsEndpointUrl}/content`,
-        user: {
```

Comment:
this confused me for quite a while.  I was copying the `user` into here, but don't need to at all since the `currentUser.js` copies the user into `res.locals` so is accessible in nunjucks directly